frontend/src/components/ConfirmationModal.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import i18n from '../common/config/i18n';
+import Button from './Button';
+import Modal from './Modal';
+
+interface ConfirmationModalProps {
+  title: string;
+  description: string;
+  onClose: () => void;
+  onConfirm: () => void;
+}
+
+const ConfirmationModal = ({ title, description, onClose, onConfirm }: ConfirmationModalProps) => {
+  return (
+    <Modal title={title} onClose={onClose}>
+      <div className="flex flex-col gap-4">
+        <p>{description}</p>
+        <div className="flex flex-row justify-end gap-3">
+          <Button
+            type="button"
+            label={i18n.t('general:cancel')}
+            className="btn-outline-secondary"
+            onClick={onClose}
+          />
+          <Button
+            type="button"
+            label={i18n.t('general:delete')}
```

Comment:
as the confirmation modall should be a generic component, why are we having the oddly specific "delete" label here?

Reply:
confirmBtnLabel: string;
  confirmBtnClassName: string;

Reply:
confirmBtnLabel: string;
  confirmBtnClassName: string;